packages/docs/stories/components/tooltip.stories.ts

```diff
@@ -0,0 +1,312 @@
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('tooltip', 'onclick'),
+      },
+    },
+  },
+  render: () => html`
+  <syn-tooltip content="Click again to dismiss" trigger="click" open>
+    <syn-button>Click to Toggle</syn-button>
+  </syn-tooltip>
+  `,
+};
+
+export const ManualTrigger: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('tooltip', 'manuel'),
+      },
+    },
+  },
+  render: () => html`
+  <syn-button style="margin-right: 4rem;">Toggle Manually</syn-button>
+  <syn-tooltip content="This is an avatar" trigger="manual" class="manual-tooltip" open>
+    <syn-icon-button name="person" label="Person" size="medium" color="neutral"></syn-icon-button>
+  </syn-tooltip>
+
+  <script type="module">
+    const tooltip = document.querySelector('.manual-tooltip');
+    const toggle = tooltip.previousElementSibling;
+
+    toggle.addEventListener('click', () => (tooltip.open = !tooltip.open));
+  </script>
+  `,
+};
+
+export const RemovingArrows: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('tooltip', 'removingarrows'),
+      },
+    },
+  },
+  render: () => html`
+  <syn-tooltip content="This is a tooltip" style="--syn-tooltip-arrow-size: 0;" open>
+    <syn-button>No Arrow</syn-button>
+  </syn-tooltip>
+  `,
+};
+
+export const HTMLInTooltips: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('tooltip', 'htmltooltip'),
+      },
+    },
+  },
+  render: () => html`
+  <syn-tooltip open>
+    <div slot="content">I'm not <strong>just</strong> a tooltip, I'm a <em>tooltip</em> with HTML!</div>
+
+    <syn-button>Hover me</syn-button>
+  </syn-tooltip>
+`,
+};
+
+export const SettingAMaximumWidth: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('tooltip', 'maxwith'),
+      },
+      story: {
+        height: '400px',
+      },
+    },
+  },
+  render: () => html`
+  <syn-tooltip style="--max-width: 80px;" content="This tooltip will wrap after only 80 pixels." open>
+    <syn-button>Hover me</syn-button>
+  </syn-tooltip>
+  `,
+};
+
+export const Hoisting: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('tooltip', 'hoisting'),
+      },
+    },
+  },
+  render: () => html`
+  <div class="tooltip-hoist">
+    <syn-tooltip content="This is a tooltip" open placement="bottom">
+      <syn-button>No Hoist</syn-button>
+    </syn-tooltip>
+
+    <syn-tooltip content="This is a tooltip" hoist open>
+      <syn-button>Hoist</syn-button>
+    </syn-tooltip>
+  </div>
+
+  <style>
+    .tooltip-hoist {
+      position: relative;
+      border: solid 2px var(--syn-panel-border-color);
+      overflow: hidden;
+      padding: var(--syn-spacing-medium);
+    }
+  </style>
+  `,
+};
+
+// Bundled screenshot story
```

Comment:
I tend to silence the warnings like this:


```suggestion
/* eslint-disable sort-keys */
```